outline/data_structures.md

```diff
@@ -53,7 +53,7 @@ you have programmed in another language, these might have been
 called arrays in that language.
 {: ng-show="block21" .description}
 
-#### Compartments like structure <button class="link" ng-model="block22" ng-click="block22=!block22">Details</button>
+#### Compartment-like structure <button class="link" ng-model="block22" ng-click="block22=!block22">Details</button>
```

Comment:
Is this really a source file? `ng-click="block22=!block22"` makes me wonder whether this file is generated from some other file.